flowtools/fcs_scatterplot/getDensityPlots.xml

```diff
@@ -1,15 +1,15 @@
-<tool id="gen_density_plots" name="Generate scatter plots" version="1.1">
+<tool id="gen_density_plots" name="Generate scatter plots" version="1.0+galaxy0">
   <description>for any given markers in FCS file</description>
   <requirements>
-    <requirement type="package" version="2.0.2">ig-flowtools</requirement>
+    <requirement type="package" version="1.10.0">bioconductor-flowdensity</requirement>
```

Comment:
is this the same bioconductor-flowdensity version that we are using on other galaxy wrappers? or is this the version advised in the newer XML from the immport repo? Ideally we want to have the same packages on the same versions to reduce the time that Galaxy spends installing coda packages.

Reply:
@pcm32 it is the recommended version in immport repo and also the same version as in other packages. So it should be fine. Immport repo also keeps flowCore as a dependency - but it seems to work without it.